lib/attribute.js

```diff
@@ -194,7 +198,7 @@ var Attribute = Inheritable.inherit({
     _emitCommit: function (branch) {
         branch = branch || this.DEFAULT_BRANCH;
         var eventString = (branch !== this.DEFAULT_BRANCH ? branch + ':' : '') + 'commit:' + this.name;
-        this.model.trigger(eventString);
+        this.model.trigger(eventString, this.model);
```

Comment:
It isn't necessary to pass a model for the trigger method. It could be better to add the model as a handler's first argument.